en/02_Developer_Guides/00_Model/02_Relations.md

```diff
@@ -989,9 +989,45 @@ class ParentObject extends DataObject
 }
 ```
 
-In this example, when the parent object is deleted, the child specified by the `has_one` relation will also
-be deleted. Note that all relation types (`has_many`, `many_many`, `belongs_many_many`, `belongs_to`, and `has_one`)
-are supported, as are methods that return lists of objects but do not correspond to a physical database relation.
+In this example, when the parent object is deleted, the child specified by the `has_one` relation will also be deleted. The relation types `has_many`, `belongs_to`, and `has_one` are supported, as are methods that return lists of objects but do not correspond to a physical database relation.
```

Comment:
This implies you cannot use `cascade_deletes` with a `many_many` relationship, which is not the case. The behaviour is different, but it is still supported.

Please update these docs so that the difference in behaviour is highlighted (perhaps with a `[!WARNING]` callout block) rather than implying that it's not supported.

Reply:
There are already a bunch of callouts in this area, so adding another will make it tough to read.

I have updated the text to hopefully make the distinction between deleting and, as an alternative, unlinking, clearer.